### DISCUSSION TITLE
Does not save VODs

### DISCUSSION THREAD

**Discussion by MartynHolland-nl** [MrBrax/LiveStreamDVR#429] [Q&A]
Hi folks,

Just found this today and am trying it out.

Seem to have it all up and running apart from all my channels are showing Does not save VODs.

Cant seem to find info on this and not had a chance to test it on any live channels yet.

Can anyone give insight into this error as i feel as if its something obvious i am overlooking

![Screenshot 2023-04-01 140028](https://user-images.githubusercontent.com/103594357/229290501-e8e79aff-5fee-4368-9bf6-d919561f4729.png)

**Comment by mr-flibble** [comment]
As far I know this is pretty normal and recording works normally.

Maybe try Check subscriptions buton whether all things are "enabled"
![image](https://user-images.githubusercontent.com/5197625/229294769-44a38f5d-45c2-40c1-829f-78e8ac2a4e2c.png)

**Comment by MrBrax** [comment]
that 'error' means that the streamer has chosen to not publish vods after they are streamed on their channel
